AlphaWallet/InCoordinator.swift

```diff
@@ -613,9 +612,9 @@ extension InCoordinator: TokensCoordinatorDelegate {
                         value: BigInt(signedOrder.order.price),
                         to: address,
                         data: Data(bytes: payload.hexa2Bytes),
-                        gasLimit: Constants.gasLimit,
+                        gasLimit: GasLimitConfiguration.max,
                         tokenId: .none,
-                        gasPrice: Constants.gasPriceDefaultERC875,
+                        gasPrice: GasLimitConfiguration.default,
```

Comment:
GasLimitConfiguration.default (`90,000`) is much lower than Constants.gasPriceDefaultERC875 was. Is it enough?

Reply:
The other one was arbitrarily high, gasPrice is generic, therefore the gas price should not distinguish between erc875 transactions and any other